x-pack/plugins/entity_manager/server/lib/entities/ingest_pipeline/generate_latest_processors.ts

```diff
@@ -30,11 +30,23 @@ function mapDestinationToPainless(metadata: MetadataField) {
 }
 
 function createMetadataPainlessScript(definition: EntityDefinition) {
-  if (!definition.metadata) {
-    return '';
-  }
+  const displayNameField = definition.displayNameTemplate.replace(/[{}]/g, '');
```

Comment:
Would this break if displayNameTemplate includes more than a single field ? something like `{{field1}} - {{field2}}`

Reply:
I'd rather explicitly setting the metadata aggregation on the definition instead of this logic since it looks hard to get right

Reply:
I agree with this, I'd rather the user has to pick a single field to use and if that field is not set we fallback to the ID.

Reply:
I didn't consider displayNameTemplate with multiple fields. I'll change this.

Reply:
@klacabane fixed here: [2bb2e1e](https://github.com/elastic/kibana/pull/201260/commits/2bb2e1e98f1163e429edf6848544e67695483fe1)